CODEOWNERS

```diff
@@ -529,7 +531,7 @@ Makefile* @cilium/build
 /pkg/health/ @cilium/sig-agent
 /pkg/hive/ @cilium/sig-foundations
 /pkg/hubble/ @cilium/sig-hubble
-/pkg/hubble/metrics @cilium/sig-hubble @cilium/sig-hubble-api
+/pkg/hubble/metrics @cilium/sig-hubble @cilium/sig-hubble-api @cilium/hubble-metrics
```

Comment:
Does it make sense to also remove `@cilium/sig-hubble` from here?

Reply:
Done.